.github/ISSUE_TEMPLATE/bug_report.md

```diff
@@ -0,0 +1,34 @@
+---
+name: Bug report
+about: Create a report to help us improve
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+**Should I file this?**
+Use this template to report *observed bugs in current Plotman*.  Do not use this to request tech support or troubleshooting; we are unable to provide that support here, so please use the user channel(s) on Keybase, or GitHub Discussions instead.  Do not use this to report issues with the core Chia plotter.  If you are using `main` branch, try pulling `development` to see if your problem is already fixed.
+
+**Describe the bug**
+A clear and concise description of what the bug is.
```

Comment:
I think it would be worth some amount of making the instructions comments.  Either the submitter sees they are comments and deletes them or they leave them alone.  Either way, we don't have to read them in the reported issue.

```suggestion
<!-- A clear and concise description of what the bug is. -->
```